tests/CLI/modules/user_tests.py

```diff
@@ -345,6 +345,18 @@ def test_devices_access(self):
         self.assert_called_with('SoftLayer_User_Customer', 'getDedicatedHosts')
         self.assert_called_with('SoftLayer_User_Customer', 'getVirtualGuests')
 
+    def test_grant_access_hardware(self):
+        result = self.run_command(['user', 'grant-access', '123456', '--hardware', '147258'])
+        self.assert_no_fail(result)
+
+    def test_grant_access_virtual(self):
+        result = self.run_command(['user', 'grant-access', '123456', '--virtual', '987456'])
+        self.assert_no_fail(result)
+
+    def test_grant_access_dedicated(self):
+        result = self.run_command(['user', 'grant-access', '123456', '--dedicated', '369852'])
+        self.assert_no_fail(result)
+
```

Comment:
Add a unit test for when a user doesn't have any device options to test the exception.

Reply:
The negative unit test was added